index.html

```diff
@@ -1,284 +1,309 @@
         </ul>
-    </div>
+      </div>
 
-    <div><hr class="black-line"></div>
+      <div>
+        <hr class="black-line">
+      </div>
 
-    <div class="vector vec3">
-      <ul>
-        <li><h4>Skills</h4></li>
-        <li><img src="/assets/Disabled.svg" alt=""></li>
-      </ul>
-      <div><hr class="black-line"></div>
-        </div>
-    
-      <div class="black-line-div"><hr class="black-line"></div>
-        
-    
-  </section>
-  
-  <section class="contactus">
-    <div class="contact1">
-      <h2 class="heading">Contact me</h2>
-      <p class="graph">If you have an application you are interested in developing, a feature that you need built or
-        a project that
-        needs coding. I'd love to help with it </p>
-      <form class="form" action="https://formspree.io/f/mvongkae" method="post">
-
-          
+      <div class="vector vec3">
+        <ul>
+          <li>
+            <h4>Skills</h4>
+          </li>
+          <li><img src="./assets/Disabled.svg" alt=""></li>
+        </ul>
         <div>
-              <input class="text" type="text" name="myName" placeholder="Yeremias" maxlength="30" required>
-            </div><br>
-            <div>
-              <input class="text" type="email" name="myEmail" placeholder="natajayanj14@gmail.com" required>
-            </div><br>
-            <div>
-              <textarea class="message" name="myText" maxlength="500" cols="30" rows="10"
-                placeholder="Write your message here" required></textarea>
-              </div>
-            <br>
-            <button type="submit" class="touch-button">Get in Touch</button>
-             <img class="under-line" src="./assets/underline.svg.svg" alt="">
+          <hr class="black-line">
         </div>
+      </div>
+
+      <div class="black-line-div">
+        <hr class="black-line">
+      </div>
+
+
+    </section>
+
+    </section>
+
+    <section class="contactus">
+      <div class="contact1">
+        <h2 class="heading">Contact me</h2>
+        <p class="graph">If you have an application you are interested in developing, a feature that you need built or
+          a project that
+          needs coding. I'd love to help with it </p>
+        <form class="form" action="https://formspree.io/f/mvongkae" method="post">
+
+
+          <div>
+            <input class="text" type="text" name="myName" placeholder="Yeremias" maxlength="30" required>
+          </div><br>
+          <div>
+            <input class="text" type="email" name="myEmail" placeholder="natajayanj14@gmail.com" required>
+          </div><br>
+          <div>
+            <textarea class="message" name="myText" maxlength="500" cols="30" rows="10"
+              placeholder="Write your message here" required></textarea>
+          </div>
+          <br>
+          <button type="submit" class="touch-button">Get in Touch</button>
+          <img class="under-line" src="./assets/underline.svg.svg" alt="">
+      </div>
```

Comment:
- [ ] I see we can Improve our contact form style to match the original design please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213243818-d48fbc07-c1a9-41b9-b676-c905748782cc.png)

what we have :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213243880-a0086327-5244-4640-b1fe-cfcdd1aa8198.png)

1. We can reduce the margin between the form and the paragraph
2. fix the border-radius on our input and textarea element to be 2px as stated in the design template
3. fix the width on the contact paragraph to 569px as mentioned in the template